### ISSUE TITLE
Converting the model to a classifier

### ISSUE THREAD

**Issue by ShilpaSangappa** [experiencor/keras-yolo3#112]
Hi,

I am trying to convert the yolov3 model to a classifier. 
So I replace all the detection layers to perform classification like this:
 # Layer 80 => 82
    x = _conv_block(x, [{'filter': 1024, 'kernel': 3, 'stride': 1, 'bnorm': True,  'leaky': True,  'layer_idx': 80}], do_skip=False)
    x1 = GlobalMaxPooling2D()(x)
    pred_yolo_1 = Dense(nb_class, activation='softmax')(x1)

This gives me an error:
 Dimension 0 in both shapes must be equal, but are 1 and 256. Shapes are [1,1,1024,256] and [256,512,1,1]. for 'Assign_285' (op: 'Assign') with input shapes: [1,1,1024,256], [256,512,1,1].
